anvil-python/anvil/commands/haproxy.py

```diff
@@ -90,14 +103,19 @@ def haproxy_questions() -> dict[str, questions.PromptQuestion]:
         ),
         "ssl_cert": questions.PromptQuestion(
             "Path to SSL Certificate for HAProxy (enter nothing to skip TLS)",
-            default_value="",
+            default_value=None,
             validation_function=validate_cert_file,
         ),
         "ssl_key": questions.PromptQuestion(
             "Path to private key for the SSL certificate (enter nothing to skip TLS)",
-            default_value="",
+            default_value=None,
             validation_function=validate_key_file,
         ),
+        "tls_mode": questions.PromptQuestion(
+            'TLS termination at HA Proxy ("termination"), passthrough to MAAS ("passthrough"), or no TLS ("")?',
+            default_value="",
```

Comment:
I would prefer this as it is more natural:

```suggestion
            default_value="disabled",
```